zcash_client_backend/Cargo.toml

```diff
@@ -214,6 +214,14 @@ unstable-serialization = ["dep:byteorder"]
 ## Exposes the [`data_api::scanning::spanning_tree`] module.
 unstable-spanning-tree = []
 
+## Exposes access to the lightwalletd server via TOR
+tor-lightwalletd-tonic = [
```

Comment:
This was placed at the end of the features section, which causes it to get categorised as an experimental feature in the crate docs.